### ISSUE TITLE
Cocoapods Frameworks vs Static Files

### ISSUE THREAD

**Issue by jesster2k10** [tipsi/tipsi-stripe#103]
I just installed the library in my project and I noticed that when static files are used, you get numerous errors stating things like:

` "_OBJC_CLASS_$_Stripe", referenced from: objc-class-ref in libTPSStripe.a(TPSStripeManager.o)`

So in order to solve this, you must add:

``` ruby
use_frameworks!
```

to the `Podfile` as the library itself references to the Framework opposed to the static files.
It would be handy to update the sample `Podfile` to include this too.

**Comment by isnifer** [comment]
As I know `use_frameworks!` is not required by `tipsi-stripe`

**Comment by jesster2k10** [comment]
@isnifer Yeah that was what I thought too but in the `TPSStripeManager.h` file it refrences to `<Stripe/Stripe.h>` which is a framework opposed to a static file

**Comment by dmytrozavgorodniy** [comment]
@jesster2k10 I just build an example app using react native. I install `tipsi-stripe` using `npm install --save tipsi-stripe`. Then I link tipsi-stripe module using `react-native link tipsi-stripe`. In my project I using Cocoapods. My Podfile looks like:

```
// Podfile
target 'Test' do
  pod 'Stripe', '~> 10.1.0'
end
```

When I build the app I don't get any errors. I don't use `use_frameworks! ` flag in my Podfile, so I'm using static libraries and in this case it works well.

How you setup `tipsi-stripe`?

**Comment by jesster2k10** [comment]
I did exactly how it was in the instructions, I used npm then I linked it with react native. My pod file looked like yours but I got errors saying that Xcode couldn’t find some files referenced from a Stripe framework so I went into some of the tips stripe files to take a look and it appeared it was using the Stripe framework opposed to the header.

That’s what made me choose to use frameworks as when I did the project built successfully. I did notice a missing reference to a Stripe framework in the products from the framework but I couldn’t find that anywhere on my drive.
